react-components/src/teams/styles.js

```diff
@@ -0,0 +1,27 @@
+import { palette } from "@cyverse-de/ui-lib";
+
+const styles = (theme) => ({
+    toolbar: {
+        backgroundColor: palette.lightGray,
+        "& div": {
+            marginRight: theme.spacing(2),
+        },
+        "& button": {
+            marginRight: theme.spacing(2),
+        },
+    },
+
+    teamFilter: {
+        width: theme.spacing(20),
+    },
+
+    table: {
+        height:
+            "calc(100% - " +
+            theme.mixins.toolbar["@media (min-width:600px)"].minHeight +
+            "px)",
+        overflow: "auto",
+    },
+});
+
+export default styles;
```

Comment:
Since you have all the code related to `Teams` in 1 file `Teams.js`, we can move this style into the `js` file and use the new `useStyles` hook. This way we can remove the wrapping. This can help with `debugging` when using react browser plugin. Again just a suggestion.